grant_token_request.go

```diff
@@ -21,6 +21,12 @@ type grantTokenBuilder struct {
 	opts *grantTokenOpts
 }
 
+type SpaceId string
+
+type grantTokenSumBuilder grantTokenBuilder
+
+type grantTokenObjectsBuilder grantTokenBuilder
```

Comment:
Do we really want to use the `Objects` qualifier in the name? I thought we were moving away from `Objects`

Reply:
I know I was out last week so correct me if I'm wrong. What are other SDKs doing with this naming conflict?